spec/models/manageiq/providers/redfish/physical_infra_manager/refresher_spec.rb

```diff
@@ -24,6 +24,7 @@ def assert_ems
     expect(ems.physical_servers.count).to eq(1)
     expect(ems.physical_servers.map(&:ems_ref)).to match_array([server_id])
     expect(ems.physical_server_details.count).to eq(1)
+    expect(ems.computer_systems.count).to eq(1)
```

Comment:
Should there be a simple assert for the `computer_system`?

Reply:
I did not add it because there is not much to check here, since ComputerSystem model only contains two fields: id and type of its managed entity. I can check if those fields match the server's, but this only checks if the refresh system is working as expected and has nothing to do with Redfish provider.